src/overlays/actors/ovl_Bg_Jya_Goroiwa/z_bg_jya_goroiwa.c

```diff
@@ -8,8 +14,14 @@ void BgJyaGoroiwa_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaGoroiwa_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaGoroiwa_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaGoroiwa_Draw(Actor* thisx, GlobalContext* globalCtx);
+void func_80897DF0(BgJyaGoroiwa* this, GlobalContext* globalCtx);
+void func_80897B48(BgJyaGoroiwa* this, GlobalContext* globalCtx);
+void func_808979C0(BgJyaGoroiwa* this, GlobalContext* ctx);
```

Comment:
We generally don't forward declare functions unless necessary, which isn't the case for some of these functions.
```suggestion
```